internal/engine/live_update_build_and_deployer.go

```diff
@@ -204,14 +208,15 @@ func liveUpdateInfoForStateTree(stateTree liveUpdateStateTree) (liveUpdInfo, err
 	var hotReload bool
 
 	if luInfo := iTarget.LiveUpdateInfo(); !luInfo.Empty() {
-		var skipped []string
-		fileMappings, skipped, err = build.FilesToPathMappings(filesChanged, luInfo.SyncSteps())
+		var pathsMatchingNoSync []string
+		fileMappings, pathsMatchingNoSync, err = build.FilesToPathMappings(filesChanged, luInfo.SyncSteps())
 		if err != nil {
 			return liveUpdInfo{}, err
 		}
-		if len(skipped) > 0 {
-			return liveUpdInfo{}, buildcontrol.RedirectToNextBuilderInfof("found file(s) not matching a LiveUpdate sync, so "+
-				"performing a full build. (Files: %s)", strings.Join(skipped, ", "))
+		if len(pathsMatchingNoSync) > 0 {
+			prettyPaths := ospath.FileListDisplayNames(iTarget.LocalPaths(), pathsMatchingNoSync)
+			return liveUpdInfo{}, buildcontrol.RedirectToNextBuilderInfof(
+				"found file(s) not matching any sync (files: %s)", strings.Join(prettyPaths, ", "))
```

Comment:
Silly little nit, but: do these have to be lowercase? And for my own edification, where are you setting the amount of indentation?

Reply:
I'm just using `\t`; is there a more easily modified way to do that? (I was assuming that if we wanted to style it separately, we'd have to make it its own log line with it's own field identifier...?)